### PR TITLE
Feature/Governance page empty state DAO-408

### PR DESCRIPTION
**Pull request by RakeshUP**

### CODE REVIEW

File: packages/web-app/src/pages/governance.tsx [hunk 1]
```diff
@@ -37,6 +48,39 @@ const Governance: React.FC = () => {
...
 
+  if (showEmptyState) {
+    return (
+      <>
+        <EmptyStateContainer>
+          <ImageContainer src={NoProposals} />
+          <EmptyStateHeading>
+            {t('governance.emptyState.title')}
+          </EmptyStateHeading>
+          <p className="mt-1.5">{t('governance.emptyState.subtitleLine1')}</p>
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/governance.tsx]: I think lines 1 & 2 for the subtitle are supposed to be one line with the text center-aligned. If we have them as two lines, they will wrap weirdly on smaller screens.

File: packages/web-app/src/pages/governance.tsx [hunk 2]
```diff
@@ -37,6 +48,39 @@ const Governance: React.FC = () => {
...
+          <p>
+            {t('governance.emptyState.subtitleLine2')}{' '}
+            <a className="font-bold text-primary-500">
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/governance.tsx]: Can you use the ui-component's Link here instead so we get all the focus and hover classes?

**Review by Fabricevladimir — COMMENTED** [review]
Thanks! Left a couple of comments

**Comment by WissenIstNacht** [comment]
Thanks a lot.

Basically, what Fabrice said :D

**Review by WissenIstNacht — APPROVED** [review]

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**